netutils/config/parser.py

```diff
@@ -700,7 +701,7 @@ def build_config_relationship(self):
 
         return self.config_lines
 
-    def _build_multiline_single_configuration_line(self, delimiter, prev_line):
+    def _build_multiline_single_configuration_line(self, delimiter: str, prev_line: str):  # type: ignore
```

Comment:
Resolvable through:

- Adding -> t.Optional[ConfigLine] to the function signature
- Add an explicit default return None at the end
- Checking for None in all the places this function is called from